src/main/java/com/fxgraph/edges/CorneredEdge.java

```diff
@@ -110,14 +163,22 @@ public EdgeGraphic(Graph graph, CorneredEdge edge, Orientation orientation, Stri
 			getChildren().add(group);
 		}
 
+		/**
+		 * @deprecated Redundant.
+		 * 
+		 * @see AbstractEdgeGraphic#getGroup()
+		 */
 		public Group getGroup() {
 			return group;
 		}
-
+		
+		/**
+		 * @deprecated Redundant.
+		 * 
+		 * @see AbstractEdgeGraphic#getText()
+		 */
```

Comment:
Since these are redundant and point to the parent class's fields you can just remove them. There won't be any reference conflicts.

Reply:
@Col-E done; see https://github.com/Col-E/fxgraph/pull/3/commits/eb76259d75dcf51a1f946cb3758ce1c81c1cc491.